advisories/0010.md

```diff
@@ -52,7 +52,9 @@ Check `runc` version:
 For more details please read the [MCR official installation documentation][mcr.docs]
 
 ### MCR 20.10.x 
-If you have MCR 20.10.12 (`containerd` 1.6 is used starting from 20.10.12) please upgrade `containerd.io` package to the fixed version:
+If you have EOL MCR 20.10.x we `STRONGLY RECOMMEND` to upgrade to the MCR 23.0.9-1.
+But we have an temporary option in case if the MCR version is 20.10.12 (`containerd` 1.6 is used starting from 20.10.12) or newer the CVE can be resolved by upgrade `containerd.io` package to the fixed version.
```

Comment:
Suggest change to

As a temporary solution for environments running MCR 20.10.12 or newer, the CVE can be resolved by upgrading only the `containerd.io` package to the one shipped with MCR 23.0.9-1 using the instructions below. This workaround is only possible on MCR 20.10.12 or newer because this is the first version of MCR 20.10 which consumed `containerd` 1.6."

Reply:
Done